Driver/app/src/main/java/com/alooma/driver/MainActivity.java

```diff
@@ -0,0 +1,89 @@
+package com.alooma.driver;
+
+import android.support.v7.app.ActionBarActivity;
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuItem;
+
+import android.view.View;
+import android.widget.EditText;
+
+import android.content.Context;
+import com.mixpanel.android.mpmetrics.MixpanelAPI;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class MainActivity extends ActionBarActivity {
+
+    public static final String MIXPANEL_TOKEN = "d36769c7f898550cbcadc7b659bb9647";
```

Comment:
We probably shouldn't merge your token into a public github repository. You may want to refresh your token now.